src/main/scala/sparkz/core/network/NetworkController.scala

```diff
@@ -29,7 +31,7 @@ import scala.util.Try
   */
 class NetworkController(settings: NetworkSettings,
                         peerManagerRef: ActorRef,
-                        scorexContext: ScorexContext,
+                        scorexContext: SparkzContext,
```

Comment:
We can rename the variable name to `sparkzContext` and the other four occurrences below